srcpkgs/vkBasalt/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'vkBasalt'
+pkgname=vkBasalt
+version=0.3.2.1
+revision=1
+archs="x86_64 i686"
+build_style=meson
+hostmakedepends="gcc>=9 glslang pkg-config"
+makedepends="libX11-devel"
+short_desc="Vulkan post-processing layer"
+maintainer="xolophreny <lowranker@protonmail.com>"
+license="Zlib"
+homepage="https://github.com/DadSchoorse/vkBasalt"
+distfiles="https://github.com/DadSchoorse/vkBasalt/archive/v${version}.tar.gz"
+checksum=b4e8d099f7d4b7ad963edfdd7708191496674890b6673f1c03763ad418b163a9
+
+
+do_configure() {
+	case "$XBPS_TARGET_MACHINE" in
+	x86_64)
+		meson --buildtype=release --prefix=/usr build;;
+	i686)
+		ASFLAGS=--32 CFLAGS=-m32 CXXFLAGS=-m32 PKG_CONFIG_PATH=/usr/lib32/pkgconfig \
+		meson --prefix=/usr \
+		--buildtype=release \
+		--libdir=lib32 \
+		-Dwith_json=false \
+		build;;
+	esac
+}
+do_build() {
+	ninja -C build install
+}
```

Comment:
This should be done by the build style automatically. And I don't believe this one works properly, since you aren't setting `DESTDIR`.